go/libraries/doltcore/sqle/dtablefunctions/dolt_log.go

```diff
@@ -515,41 +530,34 @@ func (ltf *LogTableFunction) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter
 	return ltf.NewDotDotLogTableFunctionRowIter(ctx, sqledb.DbData().Ddb, commits, notCommits, matchFunc, cHashToRefs, ltf.tableNames)
 }
 
-// evaluateArguments returns revisionValStrs, notRevisionValStrs, and three dot boolean.
-// It evaluates the argument expressions to turn them into values this LogTableFunction
-// can use. Note that this method only evals the expressions, and doesn't validate the values.
-func (ltf *LogTableFunction) evaluateArguments() (revisionValStrs []string, notRevisionValStrs []string, threeDot bool, err error) {
-	for _, expr := range ltf.revisionExprs {
-		valStr, err := expressionToString(ltf.ctx, expr)
-		if err != nil {
-			return nil, nil, false, err
-		}
-
-		if strings.Contains(valStr, "..") {
-			if strings.Contains(valStr, "...") {
-				refs := strings.Split(valStr, "...")
-				return refs, nil, true, nil
-			}
-			refs := strings.Split(valStr, "..")
-			return []string{refs[1]}, []string{refs[0]}, false, nil
+// parseRevisionRange processes a revision string for range syntax (.., ...)
+// Returns (revisions, notRevisions, threeDot, isRange)
+func parseRevisionRange(revisionStr string) ([]string, []string, bool, bool) {
```

Comment:
A nice feature of Go is that you can name return values. That provides more context to callers on what is being returned, and IDEs will use it to generate variable names, etc. If you add the return var names from your godocs to the return var section in your function signature, then you won't need to mention them in the docstring, and they'll actually end up being more useful.